app/controllers/submit.js

```diff
@@ -792,6 +780,19 @@ export default Ember.Controller.extend(BasicsValidations, NodeActionsMixin, Tagg
             model.set('isPublished', true);
             node.set('public', true);
 
+            if (this.get('applyLicense')) {
+                let newCopyrightHolders = [''];
+                if (this.get('basicsLicense.copyrightHolders') && this.get('basicsLicense.copyrightHolders').length) {
+                    newCopyrightHolders = this.get('basicsLicense.copyrightHolders').split(',');
+                }
+                if (node.get('nodeLicense.year') !== this.get('basicsLicense.year') || node.get('nodeLicense.copyrightHolders') !== newCopyrightHolders) {
```

Comment:
I think that would make sense. If applyLicense is true, then the license type will be applied to the node. So, either the user will be forced to have a copyright holders (replacing the node's) or copyright holders is not used for that license type. You can't have a license that shows copyright holders without filling that field.